libopenage/gamestate/generator.cpp

```diff
@@ -288,6 +289,8 @@ void Generator::add_units(GameMain &m) const {
 			m.placed_units.new_unit(*mvtype, p, tile.to_tile3().to_phys3());
 			tile.se += 1;
 			m.placed_units.new_unit(*fvtype, p, tile.to_tile3().to_phys3());
+			//tile.se += 2;
+			//m.placed_units.new_unit(*sctype, p, tile.to_tile3().to_phys3());
```

Comment:
I thought it will be nice start with a scout (mostly) like the original, but then I saw the idle animation and it doesn't have the frame pause (#748 part 2) so it looks weird. If you don't mind leave there to uncomment when the scout looks better...